app/src/main/res/layout/fragment_login.xml

```diff
@@ -156,11 +157,25 @@
                         android:hint="@string/login_password"
                         android:inputType="textPassword"
                         app:clearable="false"
-                        android:imeActionLabel="GO"
+                        android:imeActionLabel="Next"
+                        android:imeOptions="actionNext"
                         app:met_errorColor="@color/sclera_alert"
                         app:resettable="true"
                         app:showable="true" />
 
+                    <arcus.app.common.view.ScleraEditText
```

Comment:
This will leave a URL entry box on the main login screen. You can normally add the environment in the debug menu but this makes it a bit easier. If you provide a `arcusBasePlatformUrl` project property this will be populated with that value when you launch.